examples/host-guest/hg_setup.yaml

```diff
@@ -1,33 +1,33 @@
 #provide the full path of the protein PDB file
-protein_pdb: /Users/grinawap/Downloads/CDK2_fixed_nohet.pdb
+receptor_mol2: /Users/grinawap/PycharmProjects/perses/perses/data/host-guest/host.mol2
```

Comment:
This file still has hard-coded absolute paths. Can you change these to relative paths?

Reply:
Oh, oops! I thought I had fixed all of them. Sorry, changing now--good catch.